Octokit.Tests.Integration/Clients/PullRequestReviewCommentReactionsClientTests.cs

```diff
@@ -0,0 +1,151 @@
+﻿using Octokit;
+using Octokit.Tests.Integration;
+using Octokit.Tests.Integration.Helpers;
+using System.Threading.Tasks;
+using Xunit;
+
+public class PullRequestReviewCommentReactionsClientTests
+{
+    private readonly IGitHubClient _github;
+    private readonly IPullRequestReviewCommentsClient _client;
+    private readonly RepositoryContext _context;
+
+    const string branchName = "new-branch";
+    const string branchHead = "heads/" + branchName;
+    const string branchRef = "refs/" + branchHead;
+    const string path = "CONTRIBUTING.md";
+
+    public PullRequestReviewCommentReactionsClientTests()
+    {
+        _github = Helper.GetAuthenticatedClient();
+
+        _client = _github.PullRequest.Comment;
+
+        // We'll create a pull request that can be used by most tests
+        _context = _github.CreateRepositoryContext("test-repo").Result;
```

Comment:
The other tests are disposing of their `_context` explicitly  by implementing `IDisposable` but not this one.  Not sure if it's required or not but all 4 approaches should be consistent